examples/vector_databases/azuresearch/Getting_started_with_azure_cognitive_search_and_openai.ipynb

```diff
@@ -0,0 +1,624 @@
+       "1   2  https://simple.wikipedia.org/wiki/August  August   \n",
+       "2   6     https://simple.wikipedia.org/wiki/Art     Art   \n",
+       "3   8       https://simple.wikipedia.org/wiki/A       A   \n",
+       "4   9     https://simple.wikipedia.org/wiki/Air     Air   \n",
+       "\n",
+       "                                                text  \\\n",
+       "0  April is the fourth month of the year in the J...   \n",
+       "1  August (Aug.) is the eighth month of the year ...   \n",
+       "2  Art is a creative activity that expresses imag...   \n",
+       "3  A or a is the first letter of the English alph...   \n",
+       "4  Air refers to the Earth's atmosphere. Air is a...   \n",
+       "\n",
+       "                                        title_vector  \\\n",
+       "0  [0.001009464613161981, -0.020700545981526375, ...   \n",
+       "1  [0.0009286514250561595, 0.000820168002974242, ...   \n",
+       "2  [0.003393713850528002, 0.0061537534929811954, ...   \n",
+       "3  [0.0153952119871974, -0.013759135268628597, 0....   \n",
+       "4  [0.02224554680287838, -0.02044147066771984, -0...   \n",
+       "\n",
+       "                                      content_vector vector_id  \n",
+       "0  [-0.011253940872848034, -0.013491976074874401,...         0  \n",
+       "1  [0.0003609954728744924, 0.007262262050062418, ...         1  \n",
+       "2  [-0.004959689453244209, 0.015772193670272827, ...         2  \n",
+       "3  [0.024894846603274345, -0.022186409682035446, ...         3  \n",
+       "4  [0.021524671465158463, 0.018522677943110466, -...         4  "
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "article_df = pd.read_csv('../../data/vector_database_wikipedia_articles_embedded.csv')  \n",
+    "  \n",
+    "# Read vectors from strings back into a list using json.loads  \n",
+    "article_df[\"title_vector\"] = article_df.title_vector.apply(json.loads)  \n",
+    "article_df[\"content_vector\"] = article_df.content_vector.apply(json.loads)  \n",
+    "article_df['vector_id'] = article_df['vector_id'].apply(str)  \n",
+    "article_df.head()  \n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##  Create an index"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "azure-cognitive-search-vector-demo created\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Configure a search index\n",
+    "index_client = SearchIndexClient(\n",
+    "    endpoint=search_service_endpoint, credential=credential)\n",
+    "fields = [\n",
+    "    SimpleField(name=\"id\", type=SearchFieldDataType.String),\n",
+    "    SimpleField(name=\"vector_id\", type=SearchFieldDataType.String, key=True),\n",
```

Comment:
"id" and "vector_id" are both used in all the samples, I just included both to align with the rest